scripts/helpers/file-system.js

```diff
@@ -70,12 +70,17 @@ exports.removeFile = removeFile;
 
 /**
  * @param {string | Iterable<string>} directories
- * @param {function(string): boolean} predicate
+ * @param {(function(string): boolean) | null} predicate
```

Comment:
```js
/** @param {(function(string): boolean)} [predicate]
```

Reply:
That does not work:
`Type 'null' is not assignable to type '((arg0: string) => boolean) | undefined'.`